src/tainting/Dataflow_tainting.ml

```diff
@@ -811,7 +814,12 @@ let handle_taint_propagators env thing taints =
                * by side-effect. A pattern-propagator may use this to e.g. propagate taint
                * from `x` to `y` in `f(x,y)`, so that subsequent uses of `y` are tainted
                * if `x` was previously tainted. *)
-            | `Lval lval -> Lval_env.add lval_env lval taints_from_prop
+            | `Lval lval ->
+                if Option.is_some opt_propagated then
+                  lval_env |> Lval_env.add lval taints_from_prop
+                else
+                  (* If we could not find the taint to be propagated, it  *)
```

Comment:
I think you meant to add more details in the comment here. The sentence stops at "it".